README.md

```diff
@@ -35,9 +35,9 @@ This repository contains the following examples:
   - [x] [Deploy from contract](./examples/contracts/examples/deploy_from_contract.rs)
   - [x] [Interact with ABI](./examples/contracts/examples/interact_with_abi.rs)
 - [x] Layers (Middleware)
-  - [x] [Recommended layers](./examples/layers/examples/recommended_layers.rs)
-  - [x] [Nonce manager](./examples/layers/examples/nonce_layer.rs)
-  - [x] [Signature manager](./examples/layers/examples/signer_layer.rs)
+  - [x] [Nonce management filler](./examples/layers/examples/nonce_layer.rs)
+  - [x] [Recommended filler](./examples/layers/examples/recommended_layers.rs)
+  - [x] [Signer management layer](./examples/layers/examples/signer_layer.rs)
```

Comment:
@prestwich how would you prefer we refer to these (layer / filler / ?)?

Reply:
I've renamed it to `Fillers` for now, considering all layers now use that syntax